### PR TITLE
Bump Java source and target version to 1.7

### PR DESCRIPTION
**Pull request by FireBurn**
I'm not so familiar with the Ivy build system that OpenDJ 2.6.4 uses, I think this is everything that's needed to use Java 1.7 fully

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -478,7 +478,7 @@ Please:
   <target name="validjavaversion"
           depends="buildtools" >
 
-    <property name="min.java.version" value="1.6.0_10" />
+    <property name="min.java.version" value="1.7.0_10" />
```
Comment (chrisridd) [inline build.xml]: I believe there was a specific reason (JVM bug) that the minimum was specifically 6u10, so changing the minimum from 6u10 to 7u10 is probably slightly wrong. But I'm not sure what the correct value should be.

**Comment by FireBurn** [comment]
Fixed the docs, also not sure about min.java.version would "1.7" be enough?

**Comment by FireBurn** [comment]
Or we could make it java 1.7.0_80 which was the last public release of Java 1.7

**Comment by chrisridd** [comment]
I'd say for now leave it as 1.7.0_10 as at least that's syntactically correct.

**Pull request merged**